crates/subspace-archiving/tests/integration/archiver.rs

```diff
@@ -64,6 +119,41 @@ fn archiver() {
         expected_segment_index += 1;
         previous_root_block_hash = archived_segment.root_block.hash();
     }
+
+    // Add a block such that it fits in the next segment exactly
+    let block_3 = rand::random::<[u8; SEGMENT_SIZE - 2948]>();
```

Comment:
Magic number?

Reply:
Yes, it was selected precisely to test a specific rare case when block ends at the edge of the segment as the comment says